besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1284,6 +1284,11 @@ static class TxPoolOptionGroup {
           "Specifies the static node file containing the static nodes for this node to connect to")
   private final Path staticNodesFile = null;
 
+  @CommandLine.Option(
+      names = {"--rpc-max-logs-range"},
+      description = "Specifies the maximum number of logs to retrieve via RPC")
```

Comment:
is it a number of logs or a range of blocks?

Reply:
Range of blocks. Have updated the description.